scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,370 @@
+                              header=0)
+            for col in df6.columns:
+                df6[col] = df6[col].astype("str")
+            df7 = pd.concat([df6, df7], ignore_index=True)
+
+        # coverting year values to int
+        df7['Year'] = df7['Year'].astype(float).astype(int)
+
+        # dropping unwanted column
+        df7 = df7.drop(columns=['Unnamed: 0','Count_Person_Male',\
+            'Count_Person_Female'])
+
+        # making geoid uniform
+        df7['geo_ID'] = df7['geo_ID'].str.strip()
+
+        # sorting the values based on year and geoid
+        df7.sort_values(by=['Year', 'geo_ID'], ascending=True, inplace=True)
+
+        df7 = df7.replace('nan', '')
+        # writing output to final csv
+        df7.to_csv(_CODEDIR + "/output_files/final/" +
+                   "sex_race_aggregate_after.csv",
+                   index=False)
+
+        # collecting all the column headers
+        columns_of_as_is_output_files_after = df7.columns.to_list()
+
+        column_names[3] = columns_of_as_is_output_files_after
+
+    # aggregating the files which are aggregated
+    # from different geo granularity
+    # to final output csv
+    if len(geo_aggregate_output_files) > 0:
+        for i in geo_aggregate_output_files:
+            df4 = pd.read_csv(_CODEDIR + "/output_files/intermediate/" + i,
+                              header=0)
+            for col in df4.columns:
+                df4[col] = df4[col].astype("str")
+            df5 = pd.concat([df4, df5], ignore_index=True)
+
+        # coverting year values to int
+        df5['Year'] = df5['Year'].astype(float).astype(int)
+
+        # making geoid uniform
+        df5['geo_ID'] = df5['geo_ID'].str.strip()
+
+        # sorting the values based on year and geoid
+        df5.sort_values(by=['Year', 'geo_ID'], ascending=True, inplace=True)
+
+        # dropping unwanted column
+        df5 = df5.drop(columns=['Unnamed: 0','Count_Person_Male',\
+            'Count_Person_Female'])
+
+        # writing output to final csv
+        df5.to_csv(_CODEDIR + "/output_files/final/" +
+                   "sex_race_aggregate_national_after.csv",
+                   index=False)
+
+        # collecting all the column headers
+        columns_of_geo_aggregate_output_files = df5.columns.to_list()
+
+        column_names[4] = columns_of_geo_aggregate_output_files
+    return column_names
+
+
+def generate_mcf(statvar_names: list, flag1: int) -> None:
+    """
+    This method generates 3 MCF file w.r.t
+    dataframe headers and defined MCF template
+    1. sex_race.mcf : for the files which are processed as is.
+    2. sex_race_aggregate.mcf : for the files which
+        are having Count Person Male/Female aggregated.
+    3. sex_race_aggregate_state_2010_2020.mcf : for the files
+        which are aggregated from different geo granularity.
+
+    Args:
+        sv_names (list) : List of DataFrame Columns
+        flag1 (int) : flag value helps in generating
```

Comment:
Use "int" is pretty confusing and not quite readable, can we use an enum instead?

Reply:
Also, please use a more meaningful variable name than "flag1".

Reply:
Used Enum. Also, given a meaning variable name to flag1.